js/components/controlBar.js

```diff
@@ -48,6 +46,28 @@ var ControlBar = React.createClass({
     window.removeEventListener('orientationchange', this.closePopovers);
   },
 
+  /**
+   * Restores the focus of a previously selected control bar item.
+   * This is needed as a workaround because switching between play and pause states
+   * currently causes the control bar to re-render.
+   */
+  restoreFocusedControl: function() {
+    if (!this.props.controller.state.focusedControl || !this.domNode) {
+      return;
+    }
+    var control = this.domNode.querySelector('[data-focus-id="' + this.props.controller.state.focusedControl + '"]');
+
+    if (control && typeof control.focus === 'function') {
+      control.focus();
+      // If we got to this point it means that play was triggered using the spacebar
+      // (since a click would've cleared the focused element) and we need to
+      // trigger control bar auto hide
```

Comment:
Is there a way to make the spacebar flow and the click flow the same?

Reply:
Not in this case since the control bar timer is not started on click. When interacting with a mouse the timer is actually triggered on mouse leave. For keyboard controls all the interactions should auto show the controls.

Reply:
Got it.